coq/coq-syntax.el

```diff
@@ -488,6 +488,10 @@ so for the following reasons:
     ("Instance" nil "Instance #:#.\nProof.\n#Defined." t "Instance")
     ("Program Instance" "pinstance" "Program Instance [ # ] => # where \n# := #;\n# := #." t "Program\\s-+Instance")
     ("Let" "Let" "Let # : # := #." t "Let")
+    ("Local Ltac2" nil "Local Ltac2 # := #" t "Local\\s-+Ltac2")
+    ("Ltac2 Type" "ltac2 ty" "Ltac2 Type # := #" t "Ltac2 Type")
```

Comment:
BTW you may want to replace (for the last string argument) all `Ltac2 Sth` with `Ltac2\\s-+Sth` (cf. `Local\\s-+Ltac2`)

Reply:
I don't think we should encourage putting multiple spaces between multi-word vernaculars. I'm okay with doing `Local\\s-+Ltac2` for symmetry with `Local Ltac`, but otherwise I think it should be kept simple.

Reply:
OK, good point.